### PR TITLE
Feature/unnecessary features

### LINKED ISSUE #156: Article: Unnecessary functions

### PR DESCRIPTION
**Pull request by ridlees**
Closes #156 (by @zeman-j4)

## If an article related PR

- [x] I have checked that my suggestion follows the Contribution guide

- [x]  I have added which issue this PR closes

- [x] I have 0 warnings from Danger.js

- [x] If I have added images, they are in /files/ folder

## If other PR

- [ ] I have added which issue this PR closes

### CODE REVIEW

File: problems/unnecessary_features.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+*‌Uncollapsing all features Microsoft Word 2006 has to offer can easily become overwhelming*
+
+To provide you with an example, when Windows 8 was launched, Microsoft introduced to users a redesigned Start menu with interactive tiles. Windows 10 took everything to a new level when ads for random products came around. It's an exquisite example of a feature that no one asked for and doesn't help anyone achieve anything. Since people pay for their product, it is a terrible decision to annoy users with ads no one cares about.
+
+Contrary to preventing unnecessary features, your product may have already gotten a not so popular feature or it might have already outgrown some of its legacy features that used to be popular but over time the interest in them has slowly been decreasing. In such a case it is needed to pragmatically assess the situation by going through accessible data, user surveys and other tools that might help you make a decision whether or not to kill the feature.
+
+To provide you with an example of a successfully killed feature we would point your attention to Groove's removal of live chat from their product.
```
Comment (ridlees) [inline problems/unnecessary_features.md:46]:
```suggestion
To provide you with an example of a successfully killed feature we would point your attention to [Groove's removal of live chat from their product](https://www.groovehq.com/blog/discontinuing-live-chat).
```

File: problems/unnecessary_features.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+## How to Minimize Existence of Unnecessary Features
```
Comment (ridlees) [inline problems/unnecessary_features.md:48]: dots and capitalization

File: problems/unnecessary_features.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+  - user testing
+keywords:
+  - unnecessary feature
+  - feature creep
+  - killing feature
+covered_by_dx_scanner: false
+---
+# Unnecessary features
+**TL;DR**
+As every product or service ages it gets more bloated with unnecessary features from which not all of them are necessarily successful nor needed.
```
Comment (ridlees) [inline problems/unnecessary_features.md:24]: Do grammar fix :)

**Comment by ridlees** [comment]
@prokopsimek, review thanks a mili.

File: problems/unnecessary_features.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+**TL;DR**
+As every product or service ages, it gets more bloated with unnecessary features from which not all of them are necessarily successful nor needed. 
+
+Preventing what is also known as feature creep is needed as long as you plan on having a successful product that is easy to use and helps people get done what they need to do.
+
+## What are Unnecessary Features
+Necessary features are those adopted, useful, and loved. On the other side of the spectrum are the unnecessary features, the ones that only a few people ask for but who might not even use them when they get delivered.  Not only did you sink a lot of resources, but your product got more massive and more complicated. Preventing unnecessary features is what should be mostly done by product managers, yet even the development team shall have to say in this matter.
+
+You can spot an unnecessary feature by answering these (and others that you find fitting) questions negatively:
+-   Does this feature help users achieve what they hired your product for?
```
Comment (prokopsimek) [inline problems/unnecessary_features.md:32]:
```suggestion

-   Does this feature help users achieve what they hired your product for?
```

**Review by prokopsimek — APPROVED** [review]
Just one formatting issue.

**Pull request merged**